### DISCUSSION TITLE
如何重置脚本设置（在打不开界面的情况下）

### DISCUSSION THREAD

**Discussion by Wcraft233** [the1812/Bilibili-Evolved#3173] [提问 / Q&A] [answered]
之前用的预览版，换电脑的时候直接用的 Tampermonkey导出导入功能,现在预览版不显示按钮，卸载后再安装也显示为重新安装而非“安装”，导致重新安装后设置仍然为重置。
怎么解决呢？

**Comment by Geeyun-JY3** [comment, marked as answer]
禁用了 Tampermonkey 的 Trash Mode 没有

**Reply by Wcraft233** [reply]
没有，这项设置在哪里

**Reply by Geeyun-JY3** [reply]
![image](https://user-images.githubusercontent.com/49396421/159885085-128025e7-6f20-44ee-9967-34190d5a27ff.png)

**Reply by Geeyun-JY3** [reply]
其实不禁用也行，只是启用了之后得再去回收站里再删除脚本才算真正删除。
亲测如果脚本还在回收站里，再安装就会出现你所述情况。
